grpc/sendbuffer/sendbuffer.go

```diff
@@ -0,0 +1,111 @@
+// Copyright © 2017 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+// Package sendbuffer implements a buffered Client-Streaming RPC that drops the oldest messages on buffer overflow.
+package sendbuffer
+
+import (
+	"context"
+	"sync/atomic"
+
+	ttnlog "github.com/TheThingsNetwork/go-utils/log"
```

Comment:
I think we should eventually just rename `log` to `ttnlog`, but that's a different discussion